src/frontend/notelist.js

```diff
@@ -48,6 +48,10 @@ class NoteList {
         }
     }
 
+    async save(item, text) {
+        note.save(item.name, text);
```

Comment:
Use the already existing `write` method instead creating a new one.

```suggestion
        note.write(item.name, text);
```